OOO Users Enhancement (#10717)
demisto/content: Packs/ShiftManagement/ReleaseNotes/1_1_4.md

#### Scripts
##### GetUsersOnCall

> - Added a new argument *include_OOO_users* which allows the user to choose if to include in the output also users
>  that are Out Of Office. By default, Out Of Office users are not included in the output.

Reviewer: ```suggestion
- Added the new argument *include_OOO_users* which allows users to choose whether to include in the output users that are Out Of Office. By default, Out Of Office users are not included in the output.
```

> - Updated the docker image from `3.8.5.10845` to `3.9.1.14969`.

Reviewer: The convention should be `Upgraded the Docker image to demisto/pyjwt3:1.0.0.8871.`
